wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSenderMessageDetailsCell.swift

```diff
@@ -164,109 +139,119 @@ class ConversationSenderMessageDetailsCell: UIView, ConversationMessageCell {
     private func configureSubviews() {
         addSubview(avatar)
         addSubview(authorLabel)
-        addSubview(indicatorImageView)
         addSubview(dateLabel)
     }
 
     private func configureConstraints() {
 
-        [avatar, authorLabel, indicatorImageView, dateLabel].prepareForLayout()
+        [avatar, authorLabel, dateLabel].prepareForLayout()
 
         let trailingDateLabelConstraint  = dateLabel.trailingAnchor.constraint(
             equalTo: self.trailingAnchor,
             constant: -conversationHorizontalMargins.right
         )
 
-        indicatorImageView.setContentHuggingPriority(.required, for: .horizontal)
-
         self.trailingDateLabelConstraint = trailingDateLabelConstraint
         NSLayoutConstraint.activate([
             avatar.trailingAnchor.constraint(equalTo: authorLabel.leadingAnchor, constant: -12),
             authorLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: conversationHorizontalMargins.left),
-            indicatorImageView.leadingAnchor.constraint(equalTo: authorLabel.trailingAnchor, constant: 8),
 
             dateLabel.leadingAnchor.constraint(equalTo: authorLabel.trailingAnchor, constant: 8),
             trailingDateLabelConstraint,
 
             authorLabel.topAnchor.constraint(equalTo: self.topAnchor),
-            authorLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor),
+            self.bottomAnchor.constraint(greaterThanOrEqualTo: authorLabel.bottomAnchor),
+            self.bottomAnchor.constraint(greaterThanOrEqualTo: avatar.bottomAnchor),
 
             avatar.centerYAnchor.constraint(equalTo: authorLabel.firstBaselineAnchor),
-            dateLabel.firstBaselineAnchor.constraint(equalTo: authorLabel.firstBaselineAnchor),
-            indicatorImageView.centerYAnchor.constraint(equalTo: authorLabel.centerYAnchor)
+            dateLabel.firstBaselineAnchor.constraint(equalTo: authorLabel.firstBaselineAnchor)
         ])
     }
 
-    private func iconSize(for icon: StyleKitIcon) -> StyleKitIcon.Size {
-        return icon == .externalPartner ? 16 : 14
-    }
+    private func configureAuthorLabel(object: Configuration, user: UserType, fullName: String) {
```

Comment:
The `user` and the `fullName` can be taken from the `object`, so I'd suggest to remove them from the method signature.

Reply:
Done!